lib/browser/api/menu.js

```diff
@@ -47,8 +47,8 @@ Menu.prototype._init = function () {
   this.delegate = delegate
 }
 
-Menu.prototype.popup = function (options) {
-  if (options == null || typeof options !== 'object') {
+Menu.prototype.popup = function (options = {}) {
+  if (typeof options !== 'object') {
```

Comment:
We still need the `null` check here, `null` won't be defaulted and `typeof null === 'object'`